rethinkdb/content.md

```diff
@@ -0,0 +1,50 @@
+# What is RethinkDB?
+
+RethinkDB is an open-source, distributed database built to store JSON documents
+and effortlessly scale to multiple machines. It's easy to set up and learn and
+features a simple but powerful query language that supports table joins,
+groupings, aggregations, and functions.
+
+%%LOGO%%
+
+# How to use this image
+
+## start an instance with data mounted in the working directory
+
+The default CMD of the image is `rethinkdb --bind all`, so the RethinkDB daemon
+will bind to all network interfaces available to the container (by default,
+RethinkDB only accepts connections from `localhost`).
+
+```bash
+docker run --name some-rethink -v "$PWD:/data" -d rethinkdb
+```
+
+## connect to it from an application ...
+
+```bash
+docker run --name some-app --link some-rethink:rdb -d application-that-uses-rdb
+```
+
+## ... via the web admin interface ...
+
+```bash
+# browsing on the same machine:
+xdg-open "http://$(docker inspect --format \
+  '{{ .NetworkSettings.IPAddress }}' some-rethink):8080"
+
+# exposing and connecting remotely:
+docker run -d --link some-rethink:rdb -p 8080:8080 svendowideit/ambassador
```

Comment:
There are lots of things in this doc that are "orthogonal information that'd certainly be useful for every image (official or not)", like explaining how to start an application container with `--link` to connect to it. These things *are* listed in the docs for each image.

I agree that explaining that you can expose *any* ports via `ambassador`, ie. for the purpose of *ops*, would be too broad to mention in every image's documentation (especially considering that any ops configuration that could use it will require further thought to decide how it fits in). However, administering a currently-running container is a *common and straightforward* use scenario, and it's one that's liable to be otherwise misunderstood by the user (as seen by @yosifkit's comment).

If there's an issue with `ambassador` not being an official image, I think *making* it one is the obvious solution on that front.